guides/common/modules/ref_ipv6-and-ipv4-requirements.adoc

```diff
@@ -24,8 +24,10 @@ ifndef::foreman-deb,foreman-el,satellite[]
 * If you rely on content from IPv4-only networks, use an external dual-stack HTTP proxy.
 Configure {Project} to use this dual-stack (supporting both IPv4 and IPv6) HTTP proxy as the default HTTP proxy.
 endif::[]
+ifdef::katello,orcharhino,satellite[]
 * {Project} does not support configuring an HTTP proxy using a direct IPv6 address.
 Instead, configure the HTTP proxy with a FQDN that resolves to the IPv6 address.
+endif::[]
```

Comment:
I still debate if we should just drop this. https://github.com/Katello/katello/pull/11432 has been merged and AFAIK that was the only limitation. However, AFAIK nobody verified it so I'm still hesitant.